### PR TITLE
Terms and condition link changes

### PR DESCRIPTION
**Pull request by vsamarora**
Terms and condition link changes

### CODE REVIEW

File: portal/code/company-portal/src/components/termsncondition.tsx [hunk 1]
```diff
@@ -81,15 +74,27 @@ export default class Termsncondition extends React.Component {
...
             provider and seller conditions
             </Link>
           </span>
         );
       }
+
+      // private linkClick() {
```
Comment (ntruchsess) [inline portal/code/company-portal/src/components/termsncondition.tsx]: please remove comments that are for debugging only

File: portal/code/tractus-x-portal/src/components/termsncondition.tsx [hunk 2]
```diff
@@ -67,15 +74,27 @@ export default class Termsncondition extends React.Component {
...
             provider and seller conditions
             </Link>
           </span>
         );
       }
+
+      // private linkClick() {
```
Comment (ntruchsess) [inline portal/code/tractus-x-portal/src/components/termsncondition.tsx]: same here, please remove debugging-comment

**Review by ntruchsess — CHANGES_REQUESTED** [review]
see inline comments

File: portal/code/company-portal/src/components/termsncondition.tsx (hunk 1 above)
Comment (vsamarora) [reply portal/code/company-portal/src/components/termsncondition.tsx]: Removed

File: portal/code/tractus-x-portal/src/components/termsncondition.tsx (hunk 2 above)
Comment (vsamarora) [reply portal/code/tractus-x-portal/src/components/termsncondition.tsx]: removed

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**